R/module_teal.R

```diff
@@ -132,20 +96,15 @@ ui_teal <- function(id,
         )
       )
     ),
-    tags$hr(),
-    tags$footer(
-      tags$div(
-        footer,
-        teal.widgets::verbatim_popup_ui(ns("sessionInfo"), "Session Info", type = "link"),
-        br(),
-        ui_teal_lockfile(ns("lockfile")),
-        textOutput(ns("identifier"))
-      )
-    )
+    tags$hr()
   )
 }
 
 #' @rdname module_teal
+#' @param modules (`teal_modules`)
+#'   `teal_modules` object. These are the specific output modules which
+#'   will be displayed in the `teal` application. See [modules()] and [module()] for
+#'   more details.
```

Comment:
Move this documentation to the common `module_teal` docs as ui also has a `modules` argument. I know it will be rendered in common docs anyway but this is inconsistent location for docs.

Reply:
The reason I added it explicitly for the `srv_teal` is because it only accepts a `teal_modules` object, while the `ui_teal` and `init` can also accept `teal_module` object.

Reply:
I think it is inconsistent then between `ui_teal`/`srv_teal` so `ui_teal` should have `checkmate::assert_class(modules, "teal_modules)`